modules/DMZ/data_kit/data_filler.py

```diff
@@ -2,35 +2,40 @@
 import numpy as np
 
 
-#Drops rows which have fewer than threshold observations.
-#e.g. if threshold = 5, any rows with fewer than 5 observations will be removed from pandas_data.
+# Drops rows which have fewer than threshold observations.
+# e.g. if threshold = 5, any rows with fewer than 5 observations will be removed from pandas_data.
```

Comment:
Good question; We should figure out how we want the end-user docs look. I think at this level would be find as technically the end-users of this file will be developers of this project so they will go this deep. Honestly, maybe don't put too much thought into _where_ right now. Let's just all make sure we get documentation written and as we finalize the structure of this project we can move the docs to the right spot.